pkg/scheduler/framework/runtime/framework.go

```diff
@@ -907,6 +907,7 @@ func (f *frameworkImpl) RunScorePlugins(ctx context.Context, state *framework.Cy
 		pluginToNodeScores[pl.Name()] = make(framework.NodeScoreList, len(nodes))
 	}
 	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
```

Comment:
those has been included in another one: https://github.com/kubernetes/kubernetes/pull/111810/files, we just need to update those occurrences in one pr.  cc @Huang-Wei

Reply:
oh @sanposhiho you appended a new commit now which is duplicated with #111810?

I'm fine either way. It's just if we cover everything in this PR, the title and description need to be updated.

Reply:
Ohh sorry, I was not aware of #111810. 🙏 
Let me revert 39f459a.

Reply:
reverted.